src/flow/common.js

```diff
@@ -0,0 +1,18 @@
+import { createElement } from 'react'
+import { styled, setup } from 'goober'
```

Comment:
Do you have any concerns for adopting MUI based styling? I think it'd be less of work if we can use preconfigured components.

Reply:
no concern. just that this component right now is framework agnostic so a lean style solution is used

can swap to mui based once we determine to revise all base elements to be based on them